test_databaseconnect.py

```diff
@@ -0,0 +1,151 @@
+import databaseconnect
+import config
+import pytest
+import os
+import dotenv
+import mysql.connector  as mysql
+
+dotenv.load_dotenv()
```

Comment:
I'd request not using `dotenv` outside of `config.py`. `config.py` already deals with the loading of environment variables.

Reply:
Yes, makes sense